module-2/backend/week_two.md

```diff
@@ -13,8 +14,14 @@ end
 ```
 
 What are some methods you can call on `Team`? If these methods aren't defined in the class, how do you have access to them?
+  Team.all gives you access to all the teams available that have been stored in the database.
+  Team.new creates a new instance of team but doesnt save it
+  Team.save saves a created instance of Team and stores it in the database
+  Team.count counts the number of instances of team stored in database
+  Team.create creates and saves a new instance of team into the database
+  Team.find(id) grabs the first instance of team that matches the id
 
-3. Assume that in your database, a team has the following attributes: "id", "name", owner_id". How would you find the name of a team with an id of 4? Assuming your class only included the code from question 2, how could you find the owner of the same team?
+3. Assume that in your database, a team has the following attributes: "id", "name", owner_id". How would you find the name of a team with an id of 4? Assuming your class only included the code from question 2, how could you find the owner of the same team? Team.find(4) / Team.where(owner: Team.owner)
```

Comment:
That second part wouldn't quite work. Plus we're looking for an Owner, not a Team.

```
team = Team.find(4)
Owner.find(team.owner_id)
```